ideascube/serveradmin/package.py

```diff
@@ -8,26 +8,37 @@
 
 
 class MediaCenterPackage:
-    def __init__(self, working_dir, medias=None):
+    def __init__(self, working_dir, medias=None, report=None):
         self.working_dir = working_dir
-        self.medias = list(filter(self.check_media, medias or []))
+        self.report = report
+        filtered = filter(self.check_media, enumerate(medias or [], 1))
+        self.medias = [elem[1] for elem in filtered]
```

Comment:
Yes. There where already the filter construction and I stick with it. A for loop is far better.